scout/server/blueprints/variants/forms.py

```diff
@@ -115,9 +122,12 @@ class FiltersForm(VariantFiltersForm):
 class CancerFiltersForm(VariantFiltersForm):
     """Base filters for CancerFiltersForm - extends VariantsFiltersForm"""
 
-    depth = IntegerField("Depth >")
-    alt_count = IntegerField("Min alt count >")
-    control_frequency = BetterDecimalField("Control freq. <", places=2)
+    depth = IntegerField("Depth >", validators=[validators.Optional()])
+    alt_count = IntegerField("Min alt count >", validators=[validators.Optional()])
+    control_frequency = BetterDecimalField(
+        "Control freq. <", places=2, validators=[validators.Optional()]
+    )
+    vaf_frequency = BetterDecimalField("Tumor AF >", places=2, validators=[validators.Optional()])
```

Comment:
Tumor alt AF? Dunno, it does get long. Perhaps it is obvious, but it is easy to presuppose things in this context with allele pop freq, allele tumor/normal ratio and alt/ref ratio within tumor being on the picking list for confusion.

Reply:
Also, there is a slight risk of confusion to see % numbers in the list, but type fractions. But it is really the issue of the ones that wanted % in the table, not this input.

Reply:
This should be consistent with the control frequency I think, so a double. Tumor alt AF is more precise, I'll see if the label fits!

Reply:
Damn! 😆 
![image](https://user-images.githubusercontent.com/28093618/83520890-b08dcf00-a4de-11ea-84ed-2d544d0acc8b.png)

Reply:
Actually control freq (should probably be Normal freq for best disambiguation?) and tumor freq should be equal size?

Reply:
Yes, I've removed one col space from `Min alt count >`, that I renamed to 'Min alt count' (no >). Now I'll rename the normal freq